Orange/widgets/widget.py

```diff
@@ -65,20 +65,29 @@ class WidgetMetaClass(type(QDialog)):
 
     #noinspection PyMethodParameters
     # pylint: disable=bad-classmethod-argument
-    def __new__(mcs, name, bases, kwargs):
-        cls = super().__new__(mcs, name, bases, kwargs)
+    def __new__(mcs, name, bases, namespace, openclass=False, **kwargs):
+        cls = super().__new__(mcs, name, bases, namespace, **kwargs)
```

Comment:
Renaming `kwargs` to `namespace` and adding `**kwargs` is unrelated to this PR. These names were incorrect and puzzling.